src/shared/containers/TableViewerContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Modal } from 'antd';
+import { useNexusContext } from '@bbp/react-nexus';
+import * as csvParser from 'csv-string';
+
+import TableViewer from '../components/TableViewer';
+import useNotification from '../hooks/useNotification';
+
+const TableViewerContainer: React.FC<{
+  resourceUrl: string;
+  name: string;
+  orgLabel: string;
+  projectLabel: string;
+  onClickClose: () => void;
+}> = ({ resourceUrl, orgLabel, projectLabel, name, onClickClose }) => {
+  const nexus = useNexusContext();
+  const [tableData, setTableData] = React.useState<any>();
```

Comment:
Let us use string[][] instead of any for better type safety. prase method returns string[][]